upgrading/automated_upgrades.adoc

```diff
@@ -674,7 +674,7 @@ xref:customizing-node-upgrades[Customizing Node Upgrades], continue running the
 *_upgrade_nodes.yml_* playbook until all nodes are upgraded.
 // tag::automated_upgrade_after_reboot[]
 
-. After all master and node upgrades have completed, reboot all hosts.
+. After all master and node upgrades have completed, you can reboot all hosts if you required.
```

Comment:
Why would you need to reboot?

Reply:
If only openshift upgrade does not be required reboot, but some cases may be required. For instance, a user add some tasks using [Ansible hooks for upgrades](https://docs.openshift.com/container-platform/3.11/upgrading/automated_upgrades.html#upgrade-hooks) which is required host reboot for his upgrade playbooks. So I could not express "no reboot for upgrade" due to some irregular ones.